CONTRIBUTING.rst

```diff
@@ -1,461 +1,465 @@
-https://gist.github.com/audreyr/4feef90445b9680475f2
-
-Process: Issues
-----------------
-
-If an issue is a bug that needs an urgent fix, mark it for the next patch release.
-Then either fix it or mark as please-help.
-
-For other issues: encourage friendly discussion, moderate debate, offer your thoughts.
-
-New features require a +1 from 2 other core committers (besides yourself).
-
-Process: Roadmap
------------------
-
-The roadmap is https://github.com/audreyr/cookiecutter/milestones?direction=desc&sort=due_date&state=open
-
-Due dates are flexible. Core committers can change them as needed. Note that GitHub sort on them is buggy.
-
-How to number milestones:
-
-* Follow semantic versioning. See http://semver.org
-
-Milestone size:
-
-* If a milestone contains too much, move some to the next milestone.
-* Err on the side of more frequent patch releases.
-
-Process: Pull Request merging and HISTORY.rst maintenance
----------------------------------------------------------
-
-If you merge a pull request, you're responsible for updating `AUTHORS.rst` and `HISTORY.rst`
-
-When you're processing the first change after a release, create boilerplate following the existing pattern:
-
-    x.y.z (Development)
-    ~~~~~~~~~~~~~~~~~~~
-
-    The goals of this release are TODO: release summary of features
-
-    Features:
-
-    * Feature description, thanks to @contributor (#PR).
-
-    Bug Fixes:
-
-    * Bug fix description, thanks to @contributor (#PR).
-
-    Other changes:
-
-    * Description of the change, thanks to @contributor (#PR). 
-                      
-    .. _`@contributor`: https://github.com/contributor
-
-Process: Accepting Template Pull Requests
------------------------------------------
-
-#. Run the template to generate the project.
-#. Attempt to start/use the rendered project.
-#. Merge the template in.
-#. Update the history file.
-
-.. note:: Adding a template doesn't give authors credit.
-
-
-Process: Generating CONTRIBUTING.rst
--------------------------------------
-
-From the `cookiecutter` project root::
-
-    $ make contributing
-
-This will generate the following message::
-
-    rm CONTRIBUTING.rst
-    touch CONTRIBUTING.rst
-    cat docs/contributing.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/types_of_contributions.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/contributor_setup.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/contributor_guidelines.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/contributor_tips.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/core_committer_guide.rst >> CONTRIBUTING.rst
-    echo "\r\rAutogenerated from the docs via \`make contributing\`" >> CONTRIBUTING.rst
-    echo "WARNING: Don't forget to replace any :ref: statements with literal names"
-    WARNING: Don't forget to replace any :ref: statements with literal names

```

Comment:
Can we update this in `docs/core_committer_guide.rst` to be consistent with your change in the  `Makefile`?

Alternatively, I'm not opposed to just removing this output entirely.